converter/commands/convert.go

```diff
@@ -81,6 +85,40 @@ func convert(env environment.Environment, noRotate bool) error {
 		env.Logger,
 	)
 
+	//-------------------------------Filter setup-------------------------------
```

Comment:
Get the parsed subnets out of the config, error out on a parse error, and construct the filter